src/features/content-answers/Answer.tsx

```diff
@@ -1,20 +1,26 @@
 import * as React from 'react';
 
 import ContentAnswersGridCard from './ContentAnswersGridCard';
+import InlineError from './InlineError';
 import LoadingElement from './LoadingElement';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { ElementsXhrError } from '../../../common/types/api';
 
 import './Answer.scss';
 
 type Props = {
     answer?: string;
+    error?: ElementsXhrError;
     isLoading: boolean;
 };
 
-const Answer = ({ answer, isLoading }: Props) => {
+const Answer = ({ answer, error, isLoading }: Props) => {
     return (
         <div className="bdl-Answer">
             {answer && <ContentAnswersGridCard>{answer}</ContentAnswersGridCard>}
```

Comment:
I dont think we should give the opportunity to provide an answer and an error at the same time where it renders both

Reply:
do you suggest that only pass answer prop here and if the error is not undefined, we display InlineError directly in `ContentAnswersModalContent` component?